README.md

```diff
@@ -56,6 +56,20 @@ To run the hook over multiple files at the same time:
       files: ^requirements-dev\.(in|txt)$
 ```
 
+To check if the the lockfile is consistent with your dependencies (either with compiled requirements or toml):
```

Comment:
Since it doesn't only check consistency but actually lock dependencies, maybe this should get reworded to:
```suggestion
To lock dependencies whenever dependencies are updated:
```

Reply:
I guess I'd just say "To ensure the lockfile is up-to-date:"